__tests__/app.test.js

```diff
@@ -257,6 +257,65 @@ describe('POST/api/articles/:article_id/comments', () => {
     })
 })
 
+describe.only('PATCH/api/articles/:article_id', () => {
```

Comment:
don't forget to remove `.only` or `.skip` from your test suite once you've finished developing an endpoint 🧹